bot/cogs/common.py

```diff
@@ -178,6 +161,31 @@ async def thanks(self, ctx: Context, member: Member, *, reason: str = None) -> N
             embed.set_image(url="https://media.giphy.com/media/6tHy8UAbv3zgs/giphy.gif")
             await ctx.send(embed=embed)
 
+    @command()
+    @cooldown(1, 10, BucketType.user)
+    async def shorten(self, ctx: Context, *, link: str) -> None:
+        '''Makes a link shorter using the tinyurl api'''
```

Comment:
We use double quotes in our codebase, both in dosctrings and in regular strings when possible, fix this together with lines: 169, 173 and 177